### PR TITLE
LPS-144538 - Add text to MT bulk actions in >=lg screens

### PR DESCRIPTION
**Pull request by tinycarol**
### References

- [LPS-144538](https://issues.liferay.com/browse/LPS-144538)

### What is the goal?

* Add text labels to Management Toolbar bulk actions
* Display the labels only in >lg screens

#### Before PR
Captura de pantalla 2022-01-21 a las 10 39 59

#### After PR

https://user-images.githubusercontent.com/6642927/150504840-b9727d74-be43-422f-bfba-b0644ae22e27.mov

### How to replicate

* Enable feature flag
   * Download [this config file](https://issues.liferay.com/secure/attachment/436739/com.liferay.frontend.taglib.clay.internal.configuration.FFManagementToolbarConfiguration.config)
   * Move it to `bundles/osgi/config`
   * Restart portal
* Open browser in >lg screen size (>=1200px)
* Go to `Content & Data > Web Content`
* Create and select at least one element
* See that the bulk actions icons now have a text label
* Shrink the screen size
* See that now the text label isn't visible

### CODE REVIEW

**Review by julien — APPROVED** [review]
:muscle:

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js [hunk 1]
```diff
@@ -84,6 +92,27 @@ const ActionControls = ({
 									symbol={item.icon}
 									title={item.label}
 								/>
+
+								{showDesignImprovements && (
+									<LinkOrButton
+										className="align-items-center d-lg-inline d-none mr-2 nav-link"
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js:98]: I think we could reduce code duplication a bit, by having only one `LinkOrButton`, with its content conditionally changing. This is a minor thing, non-blocking.

**Review by markocikos — COMMENTED** [review]
Look good! Minor tweak in comments, but we can push at it is.

**Review by carloslancha — APPROVED** [review]

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js (hunk 1 above)
Comment (tinycarol) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ActionControls.js:98]: Thanks @markocikos! I started the task by doing this, but as required by other tasks in this epic, the `LinkOrButton` component already has a responsive mode where it displays either the icon or the text depending on the screen size, and adding a third mode that displays both simultaneously resulted in pretty ugly code. But I'll keep this in mind during the refactor of the MT that is in the roadmap!

**Comment by tinycarol** [comment]
ci:forward

**Pull request closed without merging**